dask/optimize.py

```diff
@@ -57,7 +58,24 @@ def cull(dsk, keys):
     return out, dependencies
 
 
-def fuse(dsk, keys=None, dependencies=None):
+def default_fused_keys_renamer(keys):
+    """Create new keys for fused tasks"""
+    if isinstance(keys[0], (str, unicode)):
+        names = [key_split(x) for x in keys[:0:-1]]
+        names.append(keys[0])
+        return '-'.join(names)
+    elif (
+        isinstance(keys[0], tuple) and len(keys[0]) > 0 and
+        isinstance(keys[0][0], (str, unicode))
+    ):
```

Comment:
Nit: please reformat to something like:

```python
    elif (isinstance(keys[0], tuple) and len(keys[0]) > 0 and
          isinstance(keys[0][0], (str, unicode))):
```

as is it took me a few seconds to realize this was still proper python :).